From 5c57ebf8c226b636da57bcc25947edabc104f294 Mon Sep 17 00:00:00 2001
From: Jack Tan <jack.tan@windriver.com>
Date: Wed, 19 May 2010 23:57:28 +0800
Subject: [PATCH 20/38] nlm_xls_atx_64_be: fixed o32 syscall

Fixed the O32 syscall compat_sys_recv. This function does not exist.

Signed-off-by: Jack Tan <jack.tan@windriver.com>
---
 arch/mips/kernel/scall64-o32.S |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/kernel/scall64-o32.S b/arch/mips/kernel/scall64-o32.S
index 1201738..105d2c6 100644
--- a/arch/mips/kernel/scall64-o32.S
+++ b/arch/mips/kernel/scall64-o32.S
@@ -379,7 +379,7 @@ EXPORT(syso32_call_table)
 	PTR	sys_getsockname
 	PTR	sys_getsockopt
 	PTR	sys_listen
-	PTR	compat_sys_recv			/* 4175 */
+	PTR	sys_recv			/* 4175 */
 	PTR	compat_sys_recvfrom
 	PTR	compat_sys_recvmsg
 	PTR	sys_send
-- 
1.6.5.2

